.github/workflows/dev_builds.yml

```diff
@@ -9,6 +9,9 @@ on:
 env:
   SIDECAR_IMAGE: "ghcr.io/sumologic/tailing-sidecar"
   OPERATOR_IMAGE: "ghcr.io/sumologic/tailing-sidecar-operator"
+  SIDECAR_IMAGE_ECR: "public.ecr.aws/sumologic/tailing-sidecar-dev"
+  OPERATOR_IMAGE_ECR: "public.ecr.aws/sumologic/tailing-sidecar-operator-dev"
+  ECR_URL: public.ecr.aws/sumologic
```

Comment:
Is it possible to build `SIDECAR_IMAGE_ECR` and `OPERATOR_IMAGE_ECR` using `ECR_URL` to not duplicate `public.ecr.aws/sumologic`?

Reply:
Now I don't know :joy: 

I though more like this

```
  ECR_URL: public.ecr.aws/sumologic
  SIDECAR_IMAGE_ECR: "${{ env.ECR_URL == 'Monday' }}/sumologic/tailing-sidecar-dev"
  OPERATOR_IMAGE_ECR: "${{ env.ECR_URL == 'Monday' }}/sumologic/tailing-sidecar-operator-dev"
```

However I don't know if such syntax is possible